concepticondata/concepticon.tsv

```diff
@@ -670,22 +670,22 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 669	RODENT	Animals	Any of the relatively small placental mammals that constitute the order Rodentia, having constantly growing incisor teeth specialized for gnawing.	Person/Thing	
 670	ROOT	Agriculture and vegetation	The absorbing and anchoring organ of a vascular plant; it bears neither leaves nor flowers and is usually subterranean.	Person/Thing	
 671	SAND	The physical world	A loose material consisting of small mineral particles, or rock and mineral particles, distinguishable by the naked eye; grains vary from almost spherical to angular, with a diameter range from 1/16 to 2 millimeters.	Person/Thing	
-672	EARLY	Time	Before the expected time.	Other	
+672	EARLY	Time	Towards the end of a time period or before the expected time.	Property	
```

Comment:
Yes, this is in fact crucial: when distinguishing things, to make a decision between, say, "child (descendant)" and "child (young human)", we NEED a list that makes the distinction. Although, since it's but a definition, it is not that crucial to change it as long as no re-mapping is involved.

Reply:
I don't think we have a list that includes both meanings on separate concepts. Buck includes both meanings in the same concept (but with a focus on the latter), the rest are without available definition or only refer to the latter.
And yeah, typo, will change "end" to "beginning". Or we leave it as the original definition, I don't mind either way.

Reply:
Ok, good alternative than. But if the lists tend to focus on the latter meaning, I'd swap them and have the "towards the end..." as the second one.